pepper-apis/src/main/java/org/broadinstitute/ddp/model/workflow/StaticState.java

```diff
@@ -32,6 +32,10 @@ public static StaticState participantList() {
         return new StaticState(StateType.PARTICIPANT_LIST);
     }
 
+    public static StaticState studyRedirect() {
+        return new StaticState(StateType.STUDY_REDIRECT);
+    }
```

Comment:
`study_redirect` is not a "static" state (e.g. it has associated data and not just a type). It does not belong here.